astromodels/xspec/factory.py

```diff
@@ -341,6 +341,10 @@ def get_models(model_dat_path):
 
                 raise ValueError("Illegal identifier name %s" % (par_name))
 
+            if hard_maximum<hard_maximum:
+
+                raise ValueError("Hard maximum (%s) < hard minimum (%s)" %(hard_maximum,hard_maximum))
```

Comment:
Typo again, should be `%(hard_maximum,hard_minimum)`

Reply:
Yes, of course :-)